src/commands/deployment/README.md

```diff
@@ -0,0 +1,73 @@
+# Service Introspection
+
+Service Introspection show information about Bedrock deployments:
+
+- Name of the person that changed the service
+- Time the service was changed or errored
+- Deployment state of the service is
+
+To use service introspection use the `spk deployment` commands. The following
+options and commands are supported:
+
+```
+Options:
+  -v, --verbose        Enable verbose logging
+  -h, --help           output usage information
+
+Commands:
+  get|g [options]      Get deployment(s) for a service, release environment, build Id, commit Id, or image tag.
+  onboard|o [options]  Onboard to use the service introspection tool. This will create a storage account in your subscription.
+  validate|v           Validate deployment(s) for a service, release environment, build Id, commit Id, or image tag.
+
+```
+
+## get command
+
+Get deployment(s) for a service, release environment, build Id, commit Id, or
+image tag.
+
+Usage: `spk deployment get|g [options]`
+
+```
+Options:
+  -b, --build-id <build-id>            Get deployments for a particular build Id from source repository
+  -c, --commit-id <commit-id>          Get deployments for a particular commit Id from source repository
+  -d, --deployment-id <deployment-id>  Get deployments for a particular deployment Id from source repository
+  -i, --image-tag <image-tag>          Get deployments for a particular image tag
+  -e, --env <environment>              Get deployments for a particular environment
```

Comment:
@samiyaakhtar where is the environment name taken from or defined?

Reply:
this is defined as the environment in the release pipeline, for eg. "Dev", "Staging" etc.